### ISSUE TITLE
Seems perspective is not working with emacs 26.2

### ISSUE THREAD

**Issue by ghost** [nex3/perspective-el#89]
After installed perspective from MELPA, I added `(persp-mode)` to .emacs.
I got following error when opening emacs...
```
Symbol's function definition is void: hash-table-values
```
Seems this also breaks emacs and I am unable to close emacs or switch buffer.
I got following error when I tried to switch buffer.
```
persp-complete-buffer: Wrong type argument: perspective, nil
```

**Comment by viveketic** [comment]
I don't think the issue is specific to emacs 26.2. I got same issue with emacs 25.2.2. I was able to solve the issue by adding following to `perspective.el` file. I am not sure this is the right way to fix the issue.
```
(eval-when-compile (require 'subr-x))
```
### References

- https://emacsredux.com/blog/2014/02/02/a-peek-at-emacs-24-dot-4-hash-table-keys-and-hash-table-values/

**Comment by gcv** [comment]
This should be fixed in commit 7d3c966ee.
